README.md

```diff
@@ -51,12 +51,6 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225154838-d6657aec-c76d-495e-bb19-d6f0eaf37051.png)

**[OPTIONAL]**

- [ ] It would be nice if you make all your commit messages meaningful and capitalize to make your pull request professional :+1: because now some of your commit messages are not capitalized :thinking: